README.md

```diff
@@ -1154,7 +1154,7 @@ Manages grant-based access privileges for users, wrapping the `postgresql::serve
 
 ##### `ensure`
 
-Specifies whether to grant or revoke the privilege. Default is to grant the privilege.
```

Comment:
This may require more information that stating it will create problems. Jean will give you some help writing this section or ensure to get it reviewed by Jean.

Reply:
Great update. We will get it reviewed by Jean to ensure its ok. I would add that revoke works on tables, the limit is only on revoke permissions on the database or something along them lines.